README.md

```diff
@@ -31,6 +31,8 @@ The Mobility Data Specification (**MDS**), a project of the [Open Mobility Found
 
 * The [`policy`][policy] API endpoints are intended to be implemented by regulatory agencies and consumed by mobility providers. Providers query the Policy API to get information about local rules that may affect the operation of their mobility service or which may be used to determine compliance. It was first released in October 2019. Development takes place under the guidance of the OMF's City Services Working Group.
 
+* The [`geography`][geography] API endpoints are intended to be implemented by regulatory agencies and consumed by mobility providers. Providers query the Policy API to get information about geographical regions for regulatory and other purposes. It was first released in October 2019, originally included as part of the Policy specification. Development takes place under the guidance of the OMF's City Services Working Group.
```

Comment:
With the new endpoint, the title line should now be "MDS is currently comprised of **four** distinct components"

Reply:
Yes.